files/image_config/sysmonitor/sysmonitor.py

```diff
@@ -0,0 +1,691 @@
+#!/usr/bin/python3
+
+from datetime import datetime
```

Comment:
need to package it into a proper python package, need unit test to validate the service.